src/Dommel/Get.cs

```diff
@@ -33,13 +34,16 @@ public static partial class DommelMapper
         /// <param name="connection">The connection to the database. This can either be open or closed.</param>
         /// <param name="id">The id of the entity in the database.</param>
         /// <param name="transaction">Optional transaction for the command.</param>
+        ///  <param name="cancellationToken">Optional cancellation token for the command.</param>
         /// <returns>The entity with the corresponding id.</returns>
-        public static async Task<TEntity?> GetAsync<TEntity>(this IDbConnection connection, object id, IDbTransaction? transaction = null)
+        public static async Task<TEntity?> GetAsync<TEntity>(this IDbConnection connection, object id, IDbTransaction? transaction = null, CancellationToken cancellationToken = default)
             where TEntity : class
         {
             var sql = BuildGetById(GetSqlBuilder(connection), typeof(TEntity), id, out var parameters);
             LogQuery<TEntity>(sql);
-            return await connection.QueryFirstOrDefaultAsync<TEntity>(sql, parameters, transaction);
+            //with CommandDefinition we can pass also the the cancellation token 
+            //for further informations https://github.com/DapperLib/Dapper/blob/main/Dapper/CommandDefinition.cs
```

Comment:
@DanieleSky you can remove these comments imo. The code is clear enough.

Reply:
Hi @henkmollema , you mean all comments like this or only the rows 44-45?

Reply:
I meant all the comments indeed.

Reply:
ok, I've done